blocks/amp-story/amp-story-cta-layer.js

```diff
@@ -50,7 +50,7 @@ const ALLOWED_BLOCKS = [
 export default registerBlockType(
 	'amp/amp-story-cta-layer',
 	{
-		title: __( 'AMP Story CTA Layer' ),
+		title: __( 'CTA Layer' ),
```

Comment:
Just realized that this didn't have a text domain, could you add these, too, where missing?